src/BTVNanoCommissioning/workflows/ctag_DY_valid_sf.py

```diff
@@ -198,6 +198,7 @@ def process_shift(self, events, shift_name):
                     self,
                     events[event_level],
                     events,
+                    None,
                     "nominal",
```

Comment:
> Hi, it looks like the output `array_...` directory contains two subdirectories: `nominal` and `n`. I did not check where this `n` is coming from, but is this expected?

Hi @mondalspandan thanks for checking it, no it wasn't expected, but I think this is due to `nominal` in empty event was not a list, need quick fix :p